README.md

```diff
@@ -533,6 +533,7 @@ aborted.
 * [ProtectedBranches](lib/overcommit/hook/pre_push/protected_branches.rb)
 * [RSpec](lib/overcommit/hook/pre_push/r_spec.rb)
 * [TestUnit](lib/overcommit/hook/pre_push/test_unit.rb)
+* [Brakeman](lib/overcommit/hook/pre_commit/brakeman.rb)
```

Comment:
This should be `lib/overcommit/hook/pre_push/brakeman.rb` as it is a `pre-push` hook.